### PR TITLE
M3-4739: Fix Firewall Landing ActionMenu and clean up CMR stuff

### PR DESCRIPTION
**Pull request by tiffwong**
Fix the alignment of the ActionMenu on the Firewall Landing page for mobile on top of cleaning up the CMR stuff from Firewalls

### CODE REVIEW

File: packages/manager/src/features/Firewalls/FirewallLanding/FirewallTable.tsx [hunk 1]
```diff
@@ -121,10 +119,14 @@ const FirewallTable: React.FC<CombinedProps> = props => {
...
-                            firewallRules={eachFirewall.rules}
-                            firewallStatus={eachFirewall.status}
+                            id={eachFirewall.id}
+                            label={eachFirewall.label}
+                            rules={eachFirewall.rules}
+                            status={eachFirewall.status}
+                            // These props are required
+                            tags={[]}
+                            created_dt=""
+                            updated_dt=""
```
Comment (DevDW) [inline packages/manager/src/features/Firewalls/FirewallLanding/FirewallTable.tsx:129]: Are we still using this file at all anywhere? The Firewalls Landing page is using the `EntityTable` pattern

**Review by DevDW — APPROVED** [review]

File: packages/manager/src/features/Firewalls/FirewallLanding/FirewallRow.test.tsx [hunk 2]
```diff
@@ -52,8 +55,8 @@ describe('FirewallRow', () => {
       );
       getByTestId('firewall-row-1');
       getByText(firewall.label);
-      getByText(firewall.status);
-      getByText(getRuleString(getCountOfRules(firewall.rules)));
+      getByText('Enabled');
+      // getByText(getRuleString(getCountOfRules(firewall.rules)));
```
Comment (tiffwong) [inline packages/manager/src/features/Firewalls/FirewallLanding/FirewallRow.test.tsx:59]: Not sure why this keeps failing

**Review by Jskobos — COMMENTED** [review]
I'm sorry this one slipped by me. I think something's gone awry with the conflict resolution; on /firewalls landing, no request to the API is made and I can't see my firewalls show up.

**Comment by Jskobos** [comment]
And I just realized that I redid most of this work in the pagination POC I've got up. Oops.

**Comment by tiffwong** [comment]
@Jskobos I'll close this one and make a new ticket for fixing the ActionMenu once your PR is merged

**Pull request closed without merging**